internal/common/baseclient/baseclient.go

```diff
@@ -89,6 +99,10 @@ func (c *BaseClient) GetID() shared.ClientID {
 	return c.id
 }
 
+func (c BaseClient) Initialise(ServerReadHandle) {}
```

Comment:
listen to linter

Reply:
I don't see any linter messages for this line